buildprocess/configureWebpack.js

```diff
@@ -2,7 +2,7 @@ var path = require('path');
 var StringReplacePlugin = require("string-replace-webpack-plugin");
 
 // If node-sass starts hanging, uncomment this line:
-//process.env.UV_THREADPOOL_SIZE = 128;
+process.env.UV_THREADPOOL_SIZE = 128;
```

Comment:
Did you mean to commit this?  I don't have a problem with it, just want to make sure.